rca/api_content/content.py

```diff
@@ -75,8 +80,9 @@ def parse_items_to_list(data, type):
                 + "?fields=_,first_published_at,social_image,body"
             )
 
-        response = try_and_fetch(detail)
-        data = response.json()
+        data = fetch_data(detail)
+        if not data:
+            return []
```

Comment:
If there isn't data (maybe because the fetch raised an exception), will anything be shown on the FE?

Reply:
It's all hidden if there is no data. I think there was a title left in, I'll ensure that's also hidden